test/utils/hardhatNode.ts

```diff
@@ -1,5 +1,5 @@
-import { ContractTransaction } from "ethers";
-import { network, ethers } from "hardhat";
+import { ContractTransaction, Signer, BigNumber } from "ethers";
```

Comment:
ethers should come from hardhat so to use bignumber just write ethers.BigNumber as it is imported on the next line

Reply:
yes, will do that